modules/c/examples/code_snippets/main.cpp

```diff
@@ -1707,6 +1707,40 @@ static void enable_custom_logging() {
     // end::set-custom-logging[]
 }
 
+static void console_log_sink() {
+    // tag::new-console-logging[]
+    CBLConsoleLogSink logSink {};
+    logSink.level = kCBLLogVerbose;
```

Comment:
@ElliotFrancisHunter I didn't include the domains option here. We have some discussion in the iOS PR. If you think I should include it, please let me know.